lib/hal/debug.c

```diff
@@ -15,9 +15,19 @@
 #define MARGIN         25
 #define MARGINS        50 // MARGIN*2
 
-int SCREEN_WIDTH	= 640;
-int SCREEN_HEIGHT	= 480;
-int SCREEN_BPP = 32;
+#define WHITE_32BPP  0x00FFFFFF
```

Comment:
Why did you move the color defines into the c file? I would assume they were better suited in the header. Reason for that thought is, that one might include the header and switch the color of the text. A contradiction to that is that color switching is not supported by the debugPrint interface. This might be the reason for the change?

Reply:
- Because they often conflict with other constants in other projects (people also called their macros for Xbox buttons `WHITE` and `BLACK` or red-black trees used it).
    - The new names make this less critical, but still.. why expose weird unprefixed names?
- There's no use for them in the header, as there's no API to use them with.
- It's not very beginner friendly to manually select between 32BPP / 16BPP / 15BPP variants.

= Generic macro names are not a good API, so it should not be exposed.

Adding color support to `debugPrint` should be avoided; consider the API soft-deprecated.